easybuild/easyconfigs/p/pplacer/pplacer-1.1.alpha19.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'Tarball'
+
+name = 'pplacer'
+version = '1.1.alpha19'
+
+homepage = 'http://matsen.fhcrc.org/pplacer/'
+description = """Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic
+ likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give
+ useful information about uncertainty, and to offer advanced visualization and downstream analysis."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# compiler pplacer from source requires an ancient OCaml version,
```

Comment:
@boegel Don't want to be 'that guy'. However, it should be 'compiling'...

Reply:
fixed in #7885